src/views/router_views/confirmRoStatements/confirmRoStatements.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block main_content %}
+
+<span class="govuk-caption-xl">
+    [first] [middle] [last] ({{i18n.confirm_ro_statements_caption}} [Month] [Year])
+  </span>
+
+{{ govukCheckboxes({
+  name: "RO Statements",
+  fieldset: {
+    legend: {
+      text: i18n.confirm_ro_statements_title,
+      isPageHeading: true,
+      classes: "govuk-fieldset__legend--l"
+    }
+  },
+  items: [
+    {
+      value: "has verified identity",
+      html: i18n.confirm_ro_statements_first_checkbox1 + "<strong>[first] [middle] [last]</strong>" + i18n.confirm_ro_statements_first_checkbox2
+    },
+    {
+      value: "is the RO",
+      html: "<strong>[SELECTED COMPANY]</strong>" + i18n.confirm_ro_statements_second_checkbox1 + "<strong>[first] [middle] [last]</strong>" + i18n.confirm_ro_statements_second_checkbox2
+    },
+    {
+      value: "is the RO of the business",
+      html: "<strong>[first] [middle] [last]</strong>" + i18n.confirm_ro_statements_third_checkbox + "<strong>[SELECTED COMPANY]</strong>"
+    }
+  ]
+}) }}
+
+{{ govukButton({
+          id: "confirmContinue",
+          text: i18n.confirm_ro_statements_continue_button,
+          href: Urls.PSC_VERIFIED
```

Comment:
Add the language selection to the href so it is maintained on the next page. Take a look at individual_statement.njk for reference.